src/components/Chatbot/Chatbot.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import AddIcon from '@mui/icons-material/Add';
+import { Stack, Typography } from '@mui/material';
+import { messageChatbot } from '../../api/messageChatbot';
+import { PrimaryButton } from '../Button/PrimaryButton';
+import { CenterPageContent } from '../CenterPageContent';
+import { ConversationLayout } from './ConversationLayout';
+import { Inputfield } from './InputField';
+
+export type messageType = {
```

Comment:
Types should follow the `PascalCase` naming convention and ideally wouldn't have "type" in their name. You don't name variables "errorVariable" etc.